library/Icinga/Chart/Donut.php

```diff
@@ -332,9 +356,9 @@ protected function assemble()
                         'attributes' => array(
                             'aria-label' => $labelBig . ' ' . $labelSmall,
                             'href' => $this->getLabelBigUrl() ? $this->getLabelBigUrl()->getAbsoluteUrl() : null,
-                            'class' => $this->labelBigEyeCatching
-                                ? 'donut-label-big donut-label-big-eye-catching'
-                                : 'donut-label-big'
+                            'class' => $this->labelBigState === null
+                                ? 'donut-label-big'
+                                : 'donut-label-big fg-stateful state-' . $this->labelBigState
```

Comment:
`fg-stateful` is a mixin, not a class. Use it inside the definition for `donut-label-big` instead. Also, please remove the `donut-label-big-eye-catching` rule as it's obsolete now.